go.mod

```diff
@@ -7,53 +7,70 @@ require (
+	k8s.io/api v0.19.0-beta.2
+	k8s.io/apimachinery v0.19.0-beta.2
+	k8s.io/apiserver v0.19.0-beta.2
+	k8s.io/client-go v0.19.0-beta.2
+	k8s.io/component-base v0.19.0-beta.2
+	k8s.io/csi-translation-lib v0.19.0-beta.2
 	k8s.io/klog v1.0.0
-	k8s.io/kubernetes v1.14.0
-	sigs.k8s.io/sig-storage-lib-external-provisioner v4.1.0+incompatible // indirect
-	sigs.k8s.io/sig-storage-lib-external-provisioner/v5 v5.0.0
+	k8s.io/kubernetes v1.18.0
+	sigs.k8s.io/sig-storage-lib-external-provisioner/v6 v6.0.0
 )
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.0
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.0-beta.2
 
-replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.0
+replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.0-beta.2
 
-replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.0
+replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.0-beta.2
 
-replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.0
+replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.0-beta.2
 
-replace k8s.io/code-generator => k8s.io/code-generator v0.17.1-beta.0
+replace k8s.io/code-generator => k8s.io/code-generator v0.19.0-beta.2
 
-replace k8s.io/cri-api => k8s.io/cri-api v0.17.1-beta.0
+replace k8s.io/cri-api => k8s.io/cri-api v0.19.0-beta.2
 
-replace k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.0
+replace k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.0-beta.2
 
-replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.0
+replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.0-beta.2
 
-replace k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.0
+replace k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.0-beta.2
 
-replace k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.0
+replace k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.0-beta.2
 
-replace k8s.io/kubectl => k8s.io/kubectl v0.17.0
+replace k8s.io/kubectl => k8s.io/kubectl v0.19.0-beta.2
 
-replace k8s.io/kubelet => k8s.io/kubelet v0.17.0
+replace k8s.io/kubelet => k8s.io/kubelet v0.19.0-beta.2
 
-replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.0
+replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.0-beta.2
 
-replace k8s.io/metrics => k8s.io/metrics v0.17.0
+replace k8s.io/metrics => k8s.io/metrics v0.19.0-beta.2
 
 replace k8s.io/node-api => k8s.io/node-api v0.17.0
 
-replace k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.0
+replace k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.19.0-beta.2
 
-replace k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.17.0
+replace k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.19.0-beta.2
 
-replace k8s.io/sample-controller => k8s.io/sample-controller v0.17.0
+replace k8s.io/sample-controller => k8s.io/sample-controller v0.19.0-beta.2
+
+replace k8s.io/api => /nvme/gopath/src/k8s.io/kubernetes/staging/src/k8s.io/api
```

Comment:
`/nvme` looks like some debugging hack.

Reply:
That's the WIP part: I need it to compile against the modified Kubernetes until the API gets merged there.

If you want to compile yourself, then you can replace this something like - beware, untested:
```
go mod edit -replace k8s.io/api=github.com/pohly/kubernetes@csi-storage-capacity
```

Reply:
Nope, that won't work. Better just check out my branch somewhere and edit the go.mod file.

Reply:
I ended up deleting go.mod completely and my editor uses stuff from `vendor/`

Reply:
The code pushed today uses officially released upstream components.